packages/blade/src/utils/index.ts

```diff
@@ -2,6 +2,7 @@ export * from './cloneDeep';
 export * from './metaAttribute';
 export * from './getColorScheme';
 export * from './getIn';
+export * from './getMediaQuery';
```

Comment:
why do we need to expose this? this will also go to consumers. do we need to export it this way?

Reply:
Seems like a good tiny utility to me. Don't see any harm in exposing.